test/controllers/auctions_controller_test.rb

```diff
@@ -0,0 +1,76 @@
+require 'test_helper'
+require 'date'
+
+class AuctionsControllerTest < ActionDispatch::IntegrationTest
+  test 'should get /auctions endpoint' do
+    get '/auctions'
+    assert_response :success
+  end
+
+  test '/auctions should retrieve all auctions as json' do
+    auctions_to_create = [
+      {
+        title: 'Mustang',
+        description: '1968 Ford Mustang',
+        start_bid: 12000.00,
+        current_bid: 12000.00,
+        end_time: 10.days.from_now,
+      },
+      {
+        title: 'Fallen Empires Booster',
+        description: '3x MTG Fallen Empires booster packs',
+        start_bid: 10.00,
+        current_bid: 14.50,
+        end_time: 10.days.from_now,
+      }
+    ]
+
+    auctions_to_create.each do |auction| Auction.create(
```

Comment:
Same as with the seed files, why don't give `create` the hash that you have instead of repeating the properties.